packages/aztec.js/test/proof/joinSplitFluid/verifier.js

```diff
@@ -1,112 +1,112 @@
-                const proof = new MintProof(currentMintCounterNote, newMintCounterNote, mintedNotes, sender);
-
-                proof.data = [proof.data[0]];
-
-                const verifier = new MintVerifier(proof);
-                const _ = verifier.verifyProof();
-                expect(verifier.isValid).to.equal(false);
-                expect(verifier.errors.length).to.equal(3);
-                expect(verifier.errors[0]).to.equal(errors.codes.SCALAR_IS_ZERO);
-                expect(verifier.errors[1]).to.equal(errors.codes.INCORRECT_NOTE_NUMBER);
-                expect(verifier.errors[2]).to.equal(errors.codes.CHALLENGE_RESPONSE_FAIL);
-                validateInputs.restore();
-            });
-        });
-    });
-
-    describe('Burn Proof Verifier', () => {
-        let burnedValues;
-        let burnedNotes;
-        let currentBurnCounter;
-        let currentBurnCounterNote;
-        let newBurnCounter;
-        let newBurnCounterNote;
-
-        before(() => {
-            currentBurnCounter = 30;
-            newBurnCounter = 50;
-            burnedValues = [10, 10];
-        });
-
-        beforeEach(async () => {
-            currentBurnCounterNote = await note.create(publicKey, currentBurnCounter);
-            newBurnCounterNote = await note.create(publicKey, newBurnCounter);
-            burnedNotes = await Promise.all(burnedValues.map((burnedValue) => note.create(publicKey, burnedValue)));
-        });
-
-        describe('Success States', () => {
-            it('should verify a valid Burn proof', async () => {
-                const proof = new BurnProof(currentBurnCounterNote, newBurnCounterNote, burnedNotes, sender);
-                const verifier = new BurnVerifier(proof);
-
-                const _ = verifier.verifyProof();
-                expect(verifier.isValid).to.equal(true);
-            });
-        });
-
-        describe('Failure States', () => {
-            it('should fail if number of notes supplied is less than 2', async () => {
-                const validateInputs = sinon.stub(Proof.prototype, 'validateInputs').callsFake(() => {});
-                const proof = new BurnProof(currentBurnCounterNote, newBurnCounterNote, burnedNotes, sender);
-
-                proof.data = [proof.data[0]];
-
-                const verifier = new BurnVerifier(proof);
-                const _ = verifier.verifyProof();
-                expect(verifier.isValid).to.equal(false);
-                expect(verifier.errors.length).to.equal(3);
-                expect(verifier.errors[0]).to.equal(errors.codes.SCALAR_IS_ZERO);
-                expect(verifier.errors[1]).to.equal(errors.codes.INCORRECT_NOTE_NUMBER);
-                expect(verifier.errors[2]).to.equal(errors.codes.CHALLENGE_RESPONSE_FAIL);
-                validateInputs.restore();
-            });
-        });
-    });
-});
+// const { errors } = require('@aztec/dev-utils');
```

Comment:
Do you mean for this to be commented?

Reply:
Good point, not sure how I missed that - should be uncommented. Fixed